js/components/Form.react.js

```diff
@@ -23,6 +23,7 @@ class LoginForm extends Component {
           <p className="form__error form__error--username-not-registered">This username does not exist.</p>
           <p className="form__error form__error--wrong-password">Wrong password.</p>
           <p className="form__error form__error--field-missing">Please fill out the entire form.</p>
+          <p className="form__error form__error--failed">Something wrong.</p>
```

Comment:
Lets change the error message to something a bit more friendly, maybe "Something went wrong, please try again!"?